src/git/config.go

```diff
@@ -141,9 +141,7 @@ func GetRemoteUpstreamURL() string {
 // GetURLHostname returns the hostname contained within the given Git URL.
 func GetURLHostname(url string) string {
 	hostnameRegex, err := regexp.Compile("(^[^:]*://([^@]*@)?|git@)([^/:]+).*")
-	if err != nil {
-		log.Fatal("Error compiling hostname regular expression: ", err)
-	}
+	logs.FatalOn(err, "Error compiling hostname regular expression: ", err)
```

Comment:
I don't like that error is passed in twice here. Thoughts on using https://github.com/pkg/errors and having the following methods
```go
logs.FatalOn(err)
logs.FatalOnWrap(err, message) // equivalent to log.FatalOn(errors.Wrap(err, message))
logs.FatalOnWrapf(err, format, v ...interface{}) // equivalent to logs.FatalOn(errors.Wrapf(err, format, ...v)
```